### ISSUE TITLE
Create & publish user guide

### ISSUE THREAD

**Issue by AARON-CLARK** [pharmaR/risk_assessment#147]
Since we are making an official release, we need documentation to show people how to use the app! Somewhat related to #133. Unfortunately, risk_assessment isn't a package, so I presume we can't use `pkgdown` (Whomp whomp).

**Comment by AARON-CLARK** [comment]
Just brainstorming - we'll have to introduce these topics to users:

- [ ] Logging in
- [ ] The pkg upload tab
- [ ] The sidebar
- [ ] Maintenance metrics
- [ ] Community Usage Metrics
- [ ] The pkg database screen (including admin features)

Since there are 6, maybe we could split up and each write two vignettes each?

**Comment by Robert-Krajcik** [comment]
I could work on logging in (and setting up userids)
and the pkg upload tab

**Comment by marlycormar** [comment]
This brings a good point @Robert-Krajcik. We have app users and app admins (and app developers), with different interests in the app. Should we split the documentation?
For instance, admins may want to know how to set up the app for the first time; users will not be involved in setting up the app, but are interested in running reports, making final decisions, etc; developers may need to understand the app's structure (code), which is related to #58.

**Comment by Robert-Krajcik** [comment]
Yeah the users wouldn't need to know how to create and modify userids and passwords.
So that would be a separate vignette for admins.

**Comment by Robert-Krajcik** [comment]
I am thinking I can put together a guide for non-admin users,
but the pkg database screen (with admin features) would probably 
need to be a separate vignette.  Thoughts??

**Comment by AARON-CLARK** [comment]
> the pkg database screen (with admin features) would probably
> need to be a separate vignette. Thoughts??

Agreed. Separate vignette for admin features.

**Comment by Robert-Krajcik** [comment]
So far I have created #152 for admins on managing user ids and passwords.
I am going to create another for admins on Changing Risk Score Weights by Metric.
And then perhaps someone could write a general user guide for everyone?

**Comment by marlycormar** [comment]
Documentation:
- [ ] For admins: Managing user ids and passwords (PR open) @Robert-Krajcik 
- [ ] For admins: Managing risk score weights (Needs PR) @AARON-CLARK 
- [ ] For users: How to use the app?  @Robert-Krajcik
      - [ ] Add small video that shows an example using the app (look for video) @marlycormar 
      - [ ] Explain each tab of the app: include how to create a report.
      - [ ] Explain the apps philosophy: how to upload a package? Talk about the final decision.
 - [ ] For users: About the app (set expectations: what the app is and is not). Maybe on the README. @marlycormar

**Comment by AARON-CLARK** [comment]
Before closing this issue, I think the user guide could benefit from a **Deployments** vignette.
